api/ProgressEvent.json

```diff
@@ -94,6 +93,45 @@
           }
         }
       },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": "38",
+              "partial_implementation": true,
+              "notes": "Available in dedicated workers and shared workers, not available in service workers. See <a href='https://crbug.com/332663431'>bug 332663431</a>."
+            },
+            "chrome_android": "mirror",
+            "deno": {
+              "version_added": "1.3"
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "54"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": "10"
+            },
```

Comment:
IE did [not support](https://github.com/mdn/browser-compat-data/blob/910d82161fde599a1c346a82db51eba3cadeb724/api/Navigator.json#L4337-L4339) service workers, and we should just omit it here.

```suggestion
```

Reply:
per https://github.com/mdn/browser-compat-data/actions/runs/11281868035/job/31378077103, this will added back and marked as not supported